src/styles/components/_component-card.scss

```diff
@@ -1,20 +1,58 @@
-.component-link {
-  display: block;
-  outline: none;
+.component-card {
+  display: flex;
+  flex-direction: column-reverse;
+  height: 21.875rem; // 350px/16
   width: 100%;
-  height: 100%;
+  cursor: pointer;
+  transition: all 150ms ease-in-out;
+  border: 2px solid var(--foreground-color);
+  background-color: var(--background-color);
 
-  .component-card {
-    display: flex;
-    flex-direction: column-reverse;
-    height: 21.875rem; // 350px/16
+  .component-link:focus & {
+    outline: none;
+    border: 2px solid var(--accent-color);
+  }
+
+  &__image {
+    box-sizing: border-box;
+    transition: all 150ms ease-in-out;
     width: 100%;
-    cursor: pointer;
+    height: 70%;
+    padding: min(15%, 2rem);
+    margin: auto;
+  }
+
+  @media (prefers-color-scheme: dark) {
+    filter: invert(1);
+  }
+
+  @media (min-width: $breakpoint-scale-up) {
+    height: 100%;
+  }
+
```

Comment:
All of this can get deleted. Since you moved this functionality into the `image-open` and `image-closed` classes, we don't need these lines. Plus this class got deleted from the `ComponentCard` code anyways.